client/e2e/workflows/bceidbusiness/industry_user.spec.ts

```diff
@@ -0,0 +1,235 @@
+    await expect(
+      page.getByText(/has been received and will be reviewed./i),
+    ).toBeVisible();
+
+    pageContent = page.locator("html");
+    await happoPlaywright.screenshot(page, pageContent, {
+      component: "Select operator admin access request confirmation",
+      variant: "default",
+    });
+  });
+
+  test("Select existing operator (via CRA business number) and request non-admin access", async ({
+    page,
+  }) => {
+    let pageContent;
+    // 🛸 Navigate directly to the operator page (already tested navigating from the dashboard in the first test)
+    const selectOperatorPage = new OperatorPOM(page);
+    await selectOperatorPage.route();
+    await selectOperatorPage.urlIsCorrect();
+    await selectOperatorPage.selectByCraNumber("987654321");
+    await expect(
+      page.getByText(selectOperatorPage.confirmationMessage),
+    ).toBeVisible();
+    await selectOperatorPage.buttonYesThisIsMyOperator.click();
+    await expect(
+      page.getByText(/Looks like you do not have access to/i),
+    ).toBeVisible();
+
+    pageContent = page.locator("html");
+    await happoPlaywright.screenshot(page, pageContent, {
+      component: "Select operator existing admin message",
+      variant: "default",
+    });
+
+    await selectOperatorPage.buttonRequestAccess.click();
+    await expect(
+      page.getByText(/Your access request has been sent/i),
+    ).toBeVisible();
+
+    pageContent = page.locator("html");
+    await happoPlaywright.screenshot(page, pageContent, {
+      component: "Select operator non-admin access request confirmation",
+      variant: "default",
+    });
+  });
+
+  test("Add a new operator with parent operators", async ({ page }) => {
+    let pageContent;
+    // 🛸 Navigate directly to the operator page (already tested navigating from the dashboard in the first test)
+    const selectOperatorPage = new OperatorPOM(page);
+    await selectOperatorPage.route();
+    await selectOperatorPage.urlIsCorrect();
+    await selectOperatorPage.linkAddOperator.click();
+    await expect(page.getByText(/Operator Information/i)).toBeVisible();
+
+    pageContent = page.locator("html");
+    await happoPlaywright.screenshot(page, pageContent, {
+      component: "Add a new operator",
+      variant: "default",
+    });
+
+    await checkRequiredFieldValidationErrors(
+      page,
+      selectOperatorPage.buttonSubmit,
+    );
+
+    pageContent = page.locator("html");
+    await happoPlaywright.screenshot(page, pageContent, {
+      component: "Add a new operator",
+      variant: "required errors",
+    });
+
+    await triggerFormatValidationErrors(page, selectOperatorPage.buttonSubmit);
+
+    pageContent = page.locator("html");
+    await happoPlaywright.screenshot(page, pageContent, {
+      component: "Add a new operator",
+      variant: "format errors",
+    });
+
+    // Fill all operator form fields
+    await fillAllFormFields(page, "fieldset#root");
+    // Add a parent operator
```

Comment:
The card mentioned doing separate tests for with/without parent operators, but I can't think of a reason why we shouldn't do it all in one test.